roles/satellite-clone/tasks/backup_check.yml

```diff
@@ -0,0 +1,75 @@
+- name: Check for config tar file
+  stat:
+    path: '{{ backup_dir }}/config_files.tar.gz'
+    get_checksum: False
+    get_md5: False
+  register: config_data
+
+- name: Check for pgsql tar file
+  stat:
+    path: '{{ backup_dir }}/pgsql_data.tar.gz'
+    get_checksum: False
+    get_md5: False
+  register: pgsql_data
+
+- name: Check for pulp data file
+  stat:
+    path: '{{ backup_dir }}/pulp_data.tar.gz'
+    get_checksum: False
+    get_md5: False
+  register: pulp_data
+
+- name: Check for candlepin dump file
+  stat:
+    path: '{{ backup_dir }}/foreman.dump'
+    get_checksum: False
+    get_md5: False
+  register: foreman_dump
+
+- name: Check for candlepin dump file
+  stat:
+    path: "{{ backup_dir }}/candlepin.dump"
+    get_checksum: False
+    get_md5: False
+  register: candlepin_dump
+
+- name: Check for mongo backup file
+  stat:
+    path: "{{ backup_dir }}/mongo_data.tar.gz"
+    get_checksum: False
+    get_md5: False
+  register: mongo_data
+
+- name: Check for mongo dump folder
+  stat:
+    path: "{{ backup_dir }}/mongo_dump"
+    get_checksum: False
+    get_md5: False
+  register: mongo_dump
+
+- name: Fail if the config tar file is not present or not accessible
+  fail: msg="{{ backup_dir }}/config_files.tar.gz is not present or not accessible"
+  when: not config_data.stat.exists
+
+- name: Set all backup types to False
+  set_fact:
+    online_backup: False
+    standard_backup: False
+
+- name: Check for online backup
+  set_fact:
+    online_backup: True
+  when: candlepin_dump.stat.exists and foreman_dump.stat.exists and mongo_dump.stat.exists
+
+- name: Check for standard backup
+  set_fact:
+    standard_backup: True
+  when: mongo_data.stat.exists and pgsql_data.stat.exists
+
```

Comment:
What happens if due to some reason, I have files for both online and standard backup in my backup folder?

In other words, what if online_backup and standard_backup are both true?